localtest/main.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/block-vision/sui-go-sdk/signer"
 	"github.com/block-vision/sui-go-sdk/sui"
 	"github.com/block-vision/sui-go-sdk/utils"
+
+	signer2 "github.com/brewmaster012/sui-gateway/signer"
```

Comment:
Typo? This is a non existing repo for me

Reply:
the module name is unfortunately github.com/brewmaster012/sui-gateway, different from the repo name. 
you should be able to build though, let me know if you can't